### PR TITLE
Suggestions for 2.0

### PR DESCRIPTION
**Pull request by ehsteve**

### CODE REVIEW

**Review by DanRyanIrish — APPROVED** [review]

File: sunpy-core.md [hunk 1]
```diff
@@ -6,4 +6,8 @@ Planned release by June 2019 with associated paper. The details of this release
 
 ## 2.0
 
+* Create a generic command to read in files such as `sunpy.open()` or `sunpy.read()`
```
Comment (Cadair) [inline sunpy-core.md:9]: :-1: 

I am really not in favour of this, there is way way way too much unpredictability in the data sources we support to make this not have 1001 weird and wonderful edge cases.

File: sunpy-core.md [hunk 2]
```diff
@@ -6,4 +6,8 @@ Planned release by June 2019 with associated paper. The details of this release
 
 ## 2.0
```
Comment (Cadair) [inline sunpy-core.md:7]: 2.0 is due to be released in May 2020 btw...

File: sunpy-core.md [hunk 3]
```diff
@@ -6,4 +6,8 @@ Planned release by June 2019 with associated paper. The details of this release
...
 
+* Create a generic command to read in files such as `sunpy.open()` or `sunpy.read()`
+* Move to use NDCube as base data object in Maps. NDCube object will become the default object returned if a file is not recognized as a defined base data object.
+* Provide improved spectrum support including fitting data models.
```
Comment (Cadair) [inline sunpy-core.md:11]: I think you need to elaborate on this to scope it down a little.

File: sunpy-core.md [hunk 4]
```diff
@@ -6,4 +6,8 @@ Planned release by June 2019 with associated paper. The details of this release
 
 ## 2.0
 
+* Create a generic command to read in files such as `sunpy.open()` or `sunpy.read()`
+* Move to use NDCube as base data object in Maps. NDCube object will become the default object returned if a file is not recognized as a defined base data object.
```
Comment (Cadair) [inline sunpy-core.md:10]: First sentence is great, second one not so much. There are many questions around how we want to handle >2D data once map is based on NDCube. I think we should aim for just the first one in 2.0 and then think about the extensions once it's been in the wild a little bit.

**Comment by Cadair** [comment]
I would like to suggest that we add composite map + reproject to 2.0.

File: sunpy-core.md (hunk 1 above)
Comment (ehsteve) [reply sunpy-core.md:9]: how else would we be able to support arbitrary data types with ndcube without a generic reader?

File: sunpy-core.md (hunk 2 above)
Comment (ehsteve) [reply sunpy-core.md:7]: I see these as goals? if it comes along and we don't have this implemented it moves to the next release.

**Comment by Cadair** [comment]
Also, finishing the Fido attribute discovery work should be high up the list for 2.0

File: sunpy-core.md (hunk 3 above)
Comment (ehsteve) [reply sunpy-core.md:11]: Ha! you think?! yes yes yes. good point.

File: sunpy-core.md (hunk 1 above)
Comment (Cadair) [reply sunpy-core.md:9]: I am not sure NDCube as it stands can support "arbitary data types", NDCube requires you have a WCS or gWCS object to work. Only "Map" like FITS ever really have those at the moment. I have never seen a timeseries with a WCS.

File: sunpy-core.md (hunk 4 above)
Comment (ehsteve) [reply sunpy-core.md:10]: that makes sense. I'll move it and make it more vague.

**Comment by ehsteve** [comment]
How about the map coregistration that has been discussed over email?

File: sunpy-core.md (hunk 2 above)
Comment (Cadair) [reply sunpy-core.md:7]: Then why bother splitting by release at all? It would be better to have a semi-prioritised list?

**Comment by Cadair** [comment]
I am tempted to leave that and see what happens with the AIA stuff.

File: sunpy-core.md (hunk 1 above)
Comment (ehsteve) [reply sunpy-core.md:9]: Spectral data like IRIS uses WCS. I agree that time series may not make sense. This obviously needs more discussion.

File: sunpy-core.md (hunk 2 above)
Comment (ehsteve) [reply sunpy-core.md:7]: Well that is a good point but we should have a place to let people know what are the plans for each release.

Comment (Cadair) [reply sunpy-core.md:7]: This is why I don't think PRs to a markdown doc is a good way to do this going forwards.

File: sunpy-core.md (hunk 3 above)
Comment (wafels) [reply sunpy-core.md:11]: astropy models?

**Pull request closed without merging**